src/base/static/utils/geospatial-utils.js

```diff
@@ -0,0 +1,183 @@
+import booleanPointInPolygon from "@turf/boolean-point-in-polygon";
+import buffer from "@turf/buffer";
+import pointsWithinPolygon from "@turf/points-within-polygon";
+import booleanOverlap from "@turf/boolean-overlap";
+import booleanContains from "@turf/boolean-contains";
+import flatten from "@turf/flatten";
+import hash from "object-hash";
+import { featureCollection } from "@turf/helpers";
+
+const getBufferFeature = (placeGeometry, bufferOptions) => {
+  try {
+    return buffer(placeGeometry, bufferOptions.distance, {
+      units: bufferOptions.units,
+    });
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error(e);
```

Comment:
I suggest that we start logging these errors to MixPanel. This should be a start:

```js
Mixpanel.track("Error", {
  message: "unable to getBufferFeature",
  error: e,
});
```

We can start adding this within all of our try/catch blocks. And I think the "Error" string token is suitable for all of our errors, which should make them easy to look up.

Reply:
Also, I notice that we are using a lot of try/catch blocks around these Turf.js functions. Are these truly errors, or are we relying on these errors for flow control, perhaps caused by invalid arguments?

If it's the latter, we should be validating the arguments we are passing in. I wouldn't want a bunch of false positive Mixpanel "errors" adding noise to our logs.

https://softwareengineering.stackexchange.com/questions/189222/are-exceptions-as-control-flow-considered-a-serious-antipattern-if-so-why

Reply:
Mixpanel tracking here sounds good. The try/catch blocks are meant to catch errors from within the turf methods. I though it would be good to have a layer of security in place since we'll be running analyses against third-party data sources, and I was nervous there might be situations in the wild which would crash turf methods. 

If any of the `catch` blocks run we probably want to know about it.